lib/dash/dash_parser.js

```diff
@@ -1689,6 +1694,16 @@ shaka.dash.DashParser = class {
     this.setUpdateTimer_(updateDelay);
   }
 
+  /**
+   * Update now the manifest
+   *
+   * @private
+   */
+  updateNow_() {
+    console.log('updateNow');
```

Comment:
Remove it

Reply:
I'm really sorry, i push it too fast and just realized all of it too late